### PR TITLE
Replace .arg().arg()... with .sprintf, asprinf isn't availabe with Qt < 5.5

### PR DESCRIPTION
**Pull request by TheOneRing**

### CODE REVIEW

File: core/varianthandler.cpp [hunk 1]
```diff
@@ -144,18 +144,18 @@ QString VariantHandler::displayString(const QVariant &value)
     {
         const auto line = value.toLine();
         return
-            QStringLiteral("%1, %2 → %3, %4").
-            arg(line.x1()).arg(line.y1()).
-            arg(line.x2()).arg(line.y2());
+            QStringLiteral("%1, %2 → %3, %4").arg(
+                QString::number(line.x1()), QString::number(line.y1()),
+                QString::number(line.x2()), QString::number(line.y2()));
```
Comment (marc-kdab) [inline core/varianthandler.cpp]: QString::asprintf

**Comment by pasnox** [comment]
QString::sprintf is non static and absolete api, is not QString::asprintf better suggestion ?

**Comment by TheOneRing** [comment]
QString::asprintf
--


Was only introduced in Qt5.5

**Comment by TheOneRing** [comment]
ping

**Pull request merged**